source/mesh_refinement/particle_density.cc

```diff
@@ -54,7 +54,7 @@ namespace aspect
             // Note that measure() only gives a linear approximation of the cell's volume.
             // This is unaccurate for curved cells, but the exact measure is likely not
             // important for this plugin since the cell distortion is usually small.
-            indicators(i) = n_tracers / cell->measure();
+            indicators(i) = static_cast<float>(n_tracers);
```

Comment:
Good point, I will update the comment. 

And yes this contradiction got me as well that is why I implemented it wrong the first time around. It is still using a particle density though. When you think about what happens during refinement it becomes more clear. The particle density (= particle number / volume) stays constant no matter how small the cells become and that is why in similar cases we usually scale things with the volume of the cell (as for example in the last lines of the 'density' refinement plugin that scales the density gradient with the cell volume). Now scaling the particle density with the volume of the cell would mean: (particle number / volume) * volume, and the volume cancels out. I can add this explanation to the comment as well.